topology/src/main/java/powsybl/tutorials/topology/VoltageLevelTopologyTutorial.java

```diff
@@ -0,0 +1,367 @@
+                .setId("DI6")
+                .setOpen(true)
+                .setNode1(4)
+                .setNode2(6)
+                .add();
+
+        vl1.getNodeBreakerView().newBreaker()
+                .setId("BR4")
+                .setOpen(false)
+                .setNode1(7)
+                .setNode2(8)
+                .add();
+        vl1.getNodeBreakerView().newDisconnector()
+                .setId("DI3")
+                .setOpen(true)
+                .setNode1(3)
+                .setNode2(7)
+                .add();
+        vl1.getNodeBreakerView().newDisconnector()
+                .setId("DI4")
+                .setOpen(false)
+                .setNode1(4)
+                .setNode2(7)
+                .add();
+
+        // We create a busbar coupler BR3.
+        vl1.getNodeBreakerView().newBreaker()
+                .setId("BR3")
+                .setOpen(false)
+                .setRetained(true) // Retain this breaker in the bus/breaker topology !
+                .setNode1(3)
+                .setNode2(4)
+                .add();
+
+        // We create a second voltage level vl2 which nominal voltage is 220 kV.
+        VoltageLevel vl2 = substation.newVoltageLevel()
+                .setId("VL2")
+                .setNominalV(220)
+                .setTopologyKind(TopologyKind.NODE_BREAKER)
+                .add();
+        vl2.getNodeBreakerView().setNodeCount(4);
+
+        // We create a busbar section BBS3.
+        vl2.getNodeBreakerView().newBusbarSection()
+                .setId("BBS3")
+                .setNode(2)
+                .add();
+
+        // We create a dangling line LN.
+        vl2.newDanglingLine()
+                .setId("LN")
+                .setNode(1)
+                .setP0(40)
+                .setQ0(0)
+                .setR(0)
+                .setX(0)
+                .setB(0)
+                .setG(0)
+                .add();
+        vl2.getNodeBreakerView().newBreaker()
+                .setId("BR6")
+                .setOpen(false)
+                .setNode1(1)
+                .setNode2(2)
+                .add();
+
+        // We create transformer TR between voltage level vl1 and voltage level vl2.
+        vl2.getNodeBreakerView().newBreaker()
+                .setId("BR5")
+                .setOpen(false)
+                .setNode1(2)
+                .setNode2(3)
+                .add();
+        substation.newTwoWindingsTransformer()
+                .setId("TR")
+                .setVoltageLevel1(vl1.getId())
+                .setNode1(8)
+                .setVoltageLevel2(vl2.getId())
+                .setNode2(3)
+                .setR(0.5)
+                .setX(10)
+                .setB(0)
+                .setG(0)
+                .setRatedU1(vl1.getNominalV())
+                .setRatedU2(vl2.getNominalV())
+                .add();
+
+        return network;
+
+    }
+
+    private VoltageLevelTopologyTutorial() {
+    }
+
+}
```

Comment:
The README.md file corresponding to the topology tutorial is missing at the moment: it seems to me there should be one even though the tutorial is also a user story. What do you think?